docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -62,36 +62,37 @@ For information about detecting the installed updates for each version of the .N
 2. In the Registry Editor, open the following subkey: **HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full**. If the **Full** subkey isn't present, then you don't have the .NET Framework 4.5 or later installed.
 
     > [!NOTE]
-    > The **NET Framework Setup** folder in the registry does not begin with a period.
+    > The **NET Framework Setup** folder in the registry does *not* begin with a period.
 
 3. Check for a DWORD entry named **Release**. If it exists, then you have .NET Framework 4.5 or later versions installed. Its value is a release key that corresponds to a particular version of the .NET Framework. In the following figure, for example, the value of the **Release** entry is *378389*, which is the release key for .NET Framework 4.5. 
 
      ![Registry entry for the .NET Framework 4.5](media/clr-installdir.png "Registry entry for the .NET Framework 4.5")
 
-The following table lists the minimum value of the **Release** entry for each .NET Framework version. You can use these values as follows:
-
-- To determine whether a minimum .NET Framework version is present, test whether the **Release** DWORD value found in the registry is *greater than or equal to* the value listed in the table. For example, if your application requires the .NET Framework 4.7 or later, you test for a minimum release key value of *460798*.
-
-- To test for multiple versions, begin with the latest .NET Framework version, and then test for each successive earlier version.
+The following table lists the value of the **Release** DWORD on individual operating systems for .NET Framework 4.5 and later versions.
 
 [!INCLUDE[Release key values note](~/includes/version-keys-note.md)]
 
 <a name="version_table"></a>
 
 |.NET Framework version|Value of the Release DWORD|
 |--------------------------------|-------------|
-|.NET Framework 4.5|378389|
-|.NET Framework 4.5.1|378675|
-|.NET Framework 4.5.2|379893|
-|.NET Framework 4.6|393295|
-|.NET Framework 4.6.1|394254|
-|.NET Framework 4.6.2|394802|
-|.NET Framework 4.7|460798|
-|.NET Framework 4.7.1|461308|
-|.NET Framework 4.7.2|461808|
-
-For a complete table of release keys for the .NET Framework for specific Windows operating system versions, see [.NET Framework release keys and Windows operating system versions](release-keys-and-os-versions.md).
+|.NET Framework 4.5|All Windows operating systems: 378389|
+|.NET Framework 4.5.1|On Windows 8.1 and Windows Server 2012 R2: 378675<br />On all other Windows operating systems: 378758|
+|.NET Framework 4.5.2|All Windows operating systems: 379893|
+|.NET Framework 4.6|On Windows 10: 393295<br />On all other Windows operating systems: 393297|
+|.NET Framework 4.6.1|On Windows 10 November Update systems: 394254<br />On all other Windows operating systems: 394271|
```

Comment:
One of the confusions we heard from customers before is if all other Windows operating systems also includes the other Windows 10 versions? Can we make that more clear?